### PR TITLE
feat: added active styles to navbar links

### PR DESCRIPTION
**Pull request by diivi**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
Adds styles for active links in the navbar
## Related Tickets & Documents
#697 

## Mobile & Desktop Screenshots/Recordings
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## Needed after this is deployed

https://github.com/open-sauced/insights/issues/1018

### CODE REVIEW

File: components/organisms/TopNav/top-nav.tsx [hunk 1]
```diff
@@ -15,31 +16,46 @@ const TopNav: React.FC = () => {
...
     <header className="top-nav-container flex justify-between items-center z-50 py-0.5 bg-light-slate-3 border-b">
       <div className="flex justify-between items-center mx-auto container px-2 md:px-16">
         <div className="flex gap-3 md:gap-8 items-center flex-wrap ">
           <HeaderLogo withBg={false} textIsBlack />
           {!!user && onboarded ? (
             <>
-              <Link className="text-sm text-light-slate-10" href={"/hub/insights"}>
+              <Link
+                className={
+                  "text-sm " + (router.asPath === "/hub/insights" ? "text-light-orange-10" : "text-light-slate-10")
```
Comment (OgDev-01) [inline components/organisms/TopNav/top-nav.tsx:30]: We can switch string concat to template literals to improve readability 
```suggestion
                  `text-sm   ${router.asPath === "/hub/insights" ? "text-light-orange-10" : "text-light-slate-10"}
```

File: components/organisms/TopNav/top-nav.tsx [hunk 2]
```diff
@@ -15,31 +16,46 @@ const TopNav: React.FC = () => {
...
+                }
+                href={"/hub/insights"}
+              >
                 Insights Hub
               </Link>
-              <Link className="text-sm text-light-slate-10" href={`/${userInterest}/dashboard/filter/recent`}>
+              <Link
+                className={
+                  "text-sm " +
```
Comment (OgDev-01) [inline components/organisms/TopNav/top-nav.tsx:38]: Applicable here as well 👍

**Review by OgDev-01 — APPROVED** [review]
Thanks for this wonderful contribution 🍕🍕🍕

Left some feedbacks 👍

**Comment by bdougie** [comment]
@diivi, please don't delete the PR template, I needed to add an issue in a section. 

There is weirdness when the user is signed in and selected "Explore." This change with the router exposes it further. I opened an issue https://github.com/open-sauced/insights/issues/1018 to make this better, but there is a full second where the menu appears to not know what to do.  

https://user-images.githubusercontent.com/5713670/225461128-6383156f-d4b4-4f31-8739-ca920f99df5c.mov

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍

File: components/organisms/TopNav/top-nav.tsx [hunk 3]
```diff
@@ -15,31 +16,45 @@ const TopNav: React.FC = () => {
...
       <div className="flex justify-between items-center mx-auto container px-2 md:px-16">
         <div className="flex gap-3 md:gap-8 items-center flex-wrap ">
           <HeaderLogo withBg={false} textIsBlack />
           {!!user && onboarded ? (
             <>
-              <Link className="text-sm text-light-slate-10" href={"/hub/insights"}>
+              <Link
+                className={`text-sm ${
+                  router.asPath === "/hub/insights" ? "text-light-orange-10" : "text-light-slate-10"
+                }`}
```
Comment (nickytonline) [inline components/organisms/TopNav/top-nav.tsx:31]: refactor (non-blocking): Consider using a function that encapsulates whether or not to use `"text-light-orange-10"` or `"text-light-slate-10"`.

```suggestion
                className={`text-sm ${
                  getActiveStyle(router.asPath === "/hub/insights")
                }`}
```

and add a function outside of the component in the file 

```javascript
function getActiveStyle(isActive) {
  return  isActive ? "text-light-orange-10" : "text-light-slate-10"
}
```

Then use this for all the top nav `<Link />` components.

Comment (diivi) [reply components/organisms/TopNav/top-nav.tsx:31]: Thanks, looks better.

**Review by bdougie — APPROVED** [review]
Need to fix the merge conflicts, but this works.

**Pull request merged**